api4/system.go

```diff
@@ -160,6 +161,34 @@ func getLogs(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.ArrayToJson(lines)))
 }
 
+func postLog(c *Context, w http.ResponseWriter, r *http.Request) {
+	if !*utils.Cfg.ServiceSettings.EnableDeveloper && !app.SessionHasPermissionTo(c.Session, model.PERMISSION_MANAGE_SYSTEM) {
```

Comment:
Yes, I asked for that specifically. I don't think we should be polluting customer logs with JS errors unless the have developer mode on, also it lets users spam the logs if they want which I think we should avoid.

Open to discussion on it though